packages/sources/polygon/src/endpoint/forex.ts

```diff
@@ -1,80 +1,147 @@
-import { Requester, Validator } from '@chainlink/ea-bootstrap'
-import { ExecuteWithConfig, Config, InputParameters } from '@chainlink/types'
+import { Requester, Validator, util } from '@chainlink/ea-bootstrap'
```

Comment:
I would probably leave this one named `tickers` because that is the last piece of the data provider's endpoint.
If there was say `forex/global`  and we used the same naming scheme it would conflict.
I could see the argument for calling this `forexTickers` too.